nvml-wrapper/src/nv_link.rs

```diff
@@ -112,6 +113,7 @@ impl<'device, 'nvml: 'device> NvLink<'device, 'nvml> {
     Supports Pascal or newer fully supported devices.
     */
     // Test written
+    #[doc(alias = "nvmlDeviceGetNvLinkVersion")]
```

Comment:
Looks like the regex missed a few methods down below here that split the `nvml_sym` call across multiple lines. Any chance it could be updated to account for that? :thinking:

Reply:
I had updated it to account for multiline calls since `device.rs` also has these, but missed executing it on the other files.

Have updated them now.